src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/AddressEntity.kt

```diff
@@ -46,22 +46,54 @@ class AddressEntity(
   @Column(name = "address_type")
   val type: String? = null,
 
+  @Column(name = "sub_building_name")
+  val subBuildingName: String? = null,
+
+  @Column(name = "building_name")
+  val buildingName: String? = null,
+
+  @Column(name = "building_number")
+  val buildingNumber: String? = null,
+
+  @Column(name = "thoroughfare_name")
+  val thoroughfareName: String? = null,
+
+  @Column(name = "dependent_locality")
+  val dependentLocality: String? = null,
```

Comment:
Are we quite sure about these names? They look like they come directly from the Postcode Address Format.
Are they at all meaningful to us, or will we purely be using them to populate from a PAF data source?